djangocms_navigation/templates/djangocms_navigation/admin/icons/base.html

```diff
@@ -9,7 +9,8 @@
     cms-versioning-action-{% block name %}{% endblock %}
     {% if get|default_if_none:True %} cms-form-get-method {% endif %}
     {% if action|default_if_none:True %} js-versioning-action{% endif %}
-    js-versioning-admin-keep-sideframe"
+    {% if keepsideframe|default_if_none:True %}js-versioning-keep-sideframe {% else %}js-versioning-admin-close-sideframe {% endif %}
```

Comment:
The close sideframe is wrong, it should be js-versioning-close-sideframe: https://github.com/django-cms/djangocms-versioning/blob/master/djangocms_versioning/static/djangocms_versioning/js/actions.js#L38

Reply:
I can't see any differences from this base file to the versioning one, unlike djangocms-pageadmin which has it's own classes and heavily reuses the core cms's styles. 

I would say it would be best to use versionings and fix the versioning base icon template which isn't using the correct class: 'js-versioning-keep-sideframe'